### ISSUE TITLE
Can't access site_admin from collaborator endpoint

### ISSUE THREAD

**Issue by boreal321** [PyGithub/PyGithub#713]
Hello,

I'm trying to determine if a collaborator is a site admin but the site_admin key does not get returned:

## code

```
for repo in g.get_organization('orgName').get_repos():
    for collab in repo.get_collaborators():
        print(collab.site_admin)
```

## error

```
Traceback (most recent call last):
  File "protected_test.py", line 27, in <module>
    print(collab.site_admin)
AttributeError: 'NamedUser' object has no attribute 'site_admin'
```

Thanks

**Comment by sfdye** [comment]
@boreal321 Thanks for reporting. `site_admin` needs to be implemented for `NamedUser`. This should be an easy fix, a PR will be welcomed.

**Comment by sfdye** [comment]
Fixed in #719
